doc/migrations.md

```diff
@@ -5,26 +5,32 @@
 If you are planning to use database migrations, you need to let
 Alembic know the current state of the database.
 
-Presuming you've just started off by running `make load_demo_data`,
+Presuming you've just started off by running `make load_demo_data`
+on the latest master branch commit (do not check out your database schema changes yet),
```

Comment:
What does "do not check out your database schema changes yet" mean?

Perhaps: "(this should happen on a vanilla master branch, i.e. *without* any of your changes to the database schema)"